exercises/perfect-numbers/perfect_numbers_test.go

```diff
@@ -0,0 +1,48 @@
+package perfect
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestPositiveNumberRequiredErrorDefined(t *testing.T) {
+	if !reflect.TypeOf(ErrOnlyPositive).Implements(reflect.TypeOf((*error)(nil)).Elem()) {
```

Comment:
Do you feel strongly about using this runtime check rather than a compile-time check of `var _ error = ErrOnlyPositive`?